microsoft-edge/webview2/how-to/webdriver.md

```diff
@@ -104,14 +104,18 @@ Microsoft Edge WebDriver launches your WebView2 app and automatically attaches t
 
 *  The "attach" approach: In other scenarios, it's appropriate to attach Microsoft Edge WebDriver to a running WebView2 instance.  You launch your app outside of Microsoft Edge WebDriver, and then attach Microsoft Edge WebDriver to a running WebView2 instance.  This "attach" approach is for a WebView2 app that's not compatible with the "launch" approach.
 
-### Approach 1: Letting Microsoft Edge WebDriver launch your WebView2 app
+
+<!-- ------------------------------ -->
```

Comment:
Is this meant to be a part of this change?

Reply:
The comment lines are added by the writer, I think he is doing some clean up.